### PR TITLE
feat: reset_fork block number argument

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What I did

fixes: #

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by unparalleled-js** [comment]
PR already approved elsewhere, just making sure build passes before merge

File: tests/cache/hardhat-network-fork/network-1/request-0a323c2abcb5d30f127be1560fe14bc5.json [hunk 1]
```diff
@@ -0,0 +1 @@
+["0x","0x330","0xadb360ce69138b26bb"]
```
Comment (unparalleled-js) [inline tests/cache/hardhat-network-fork/network-1/request-0a323c2abcb5d30f127be1560fe14bc5.json:1]: By including these cache files, we don't ever need to make requests to Alchemy.

File: tests/ape-config.yaml [hunk 2]
```diff
@@ -9,8 +9,10 @@ hardhat:
     ethereum:
       mainnet:
         upstream_provider: alchemy
+        block_number: 15000000
```
Comment (unparalleled-js) [inline tests/ape-config.yaml:12]: Specifying block number allows cache to get created which speeds things up and saves API requests:
 https://hardhat.org/hardhat-network/docs/guides/forking-other-networks#pinning-a-block

**Review by NotPeopling2day — APPROVED** [review]
LGTM

File: tests/cache/hardhat-network-fork/network-1/request-0a323c2abcb5d30f127be1560fe14bc5.json (hunk 1 above)
Comment (NotPeopling2day) [reply tests/cache/hardhat-network-fork/network-1/request-0a323c2abcb5d30f127be1560fe14bc5.json:1]: I think using these cached values will help with plugin testing and we can keep an eye toward creating the CI workflow of testing that can do more complex and 'live' calls as we need them.

**Pull request merged**